client/verta/verta/registry/action_type/_unknown.py

```diff
@@ -0,0 +1,19 @@
+from ..._protos.public.registry import RegistryService_pb2
+
+from . import _ActionType
+
+
+class Unknown(_ActionType):
+    """
+    Unknown action of the registered model.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        from verta.registry import action_type
+        reg_model = client.create_registered_model("My Model", workspace="my-org", action_type=action_type.Unknown())
+
+    """
```

Comment:
We should mark `Unknown` as private `_Unknown`, because users shouldn't be using it directly.

```suggestion
class _Unknown(_ActionType):
    """
    Unknown action of the registered model. Not for external use.

    """
```